mobile/geth.go

```diff
@@ -199,6 +280,22 @@ func (n *Node) Stop() error {
 	return n.node.Stop()
 }
 
+func (n *Node) AddPeer(enode string) error {
```

Comment:
you create an rpc client for every `AddPeer` call?
not sure what you are trying to do here, comments would help, in fact for exported methods linter will fail if you write no comment